Documentation/SwiftGenKit Contexts/MigrationGuide.md

```diff
@@ -1,9 +1,29 @@
-# Master
+# SwiftGen 6.0 Migration Guide
 
-*** Work In Progress — Will be consolidated once the release is ready ***
+## Changes for users
+
+### Strings
+
+The parser will no longer normalize string keys, which can lead to duplicate keys if your file contain similar keys but with different casing. For example, if your file contains:
+
+```
+"abcNews.something" = "foo";
+"ABCNews.somethingElse" = "bar";
+```
+
+SwiftGenKit will no longer consolidate these into one "abcNews" case. It is up to the user to fix this incorrect casing in their `strings` files.
```

Comment:
s/incorrect/inconsistent/

[…] in their `.strings` files, or to adapt their custom template to take that in consideration.